addons/calendar/static/src/activity/activity_model_patch.js

```diff
@@ -11,6 +12,25 @@ patch(Activity, {
 });
 
 patch(Activity.prototype, {
+    setup() {
+        super.setup(...arguments);
+        /** @type {luxon.DateTime} */
+        this.start = Record.attr(undefined, { type: "datetime" });
+        /** @type {luxon.DateTime} */
+        this.stop = Record.attr(undefined, { type: "datetime" });
+    },
+    get activityTime() {
+        if (this.start && this.stop) {
+            const start = this.start.toLocaleString(luxon.DateTime.TIME_24_SIMPLE);
+            const stop = this.stop.toLocaleString(luxon.DateTime.TIME_24_SIMPLE);
+            return ` (${start} - ${stop})`;
```

Comment:
You should not add the extra space over here. It should be added while displaying in template.